go.mod

```diff
@@ -1,24 +1,24 @@
 module cloud.ibm.com/cloud-provider-ibm
 
-go 1.22
+go 1.22.4
```

Comment:
This is causing issues since our toolchain is currently 1.22.1, can we carry something to bring this down?

Reply:
yeah, i can add a carry to this